frontend/src/views/Login.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="login">
+    <h3>Sign In</h3>
+    <input type="text" v-model="email" placeholder="Email" />
+    <br />
+    <input type="password" v-model="password" placeholder="Password" />
+    <br />
+    <button @click="login">Connect</button>
+    <p>
+      You don't have an account? You can
+      <router-link to="/sign-up">create one</router-link>
+    </p>
+  </div>
+</template>
+
+<script>
+import firebase from "firebase";
+export default {
+  name: "login",
+  data() {
+    return {
+      email: "",
+      password: ""
+    };
+  },
+  methods: {
+    login: function() {
+      firebase
+        .auth()
+        .signInWithEmailAndPassword(this.email, this.password)
+        .then(
+          () => {
+            this.$router.replace("home");
+            alert("Well done! You are now connected");
+          },
+          err => {
+            alert("Oops," + err.message);
+          }
+        );
+    }
```

Comment:
Feel free to ignore this comment. As the Googlers are primary user of the prototype, Can you use the Google Sign In instead of Email / Password. One tap thing works like magic!

Reply:
That is extremely true. I will try to inculcate both email and Google sign-in options. Thank you so much for reviewing my code.